packages/react/src/Bento/Bento.features.stories.tsx

```diff
@@ -0,0 +1,163 @@
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} />
+    <Bento.Item columnSpan={5} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+  </Bento>
+)
+
+export const MixedThemeItems: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} colorMode="dark" />
+    <Bento.Item columnSpan={5} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} />
+    <Bento.Item columnSpan={12} rowSpan={4} colorMode="dark" />
+  </Bento>
+)
+
+export const ResponsiveItems: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+    <Bento.Item
+      columnSpan={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
+      rowSpan={{xsmall: 1, small: 2, medium: 3, large: 4}}
+    />
+  </Bento>
+)
+
+export const ImageGallery: StoryFn<typeof Bento> = () => (
+  <Bento>
+    <Bento.Item columnSpan={7} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={5} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+    <Bento.Item columnSpan={12} rowSpan={4} visualAsBackground>
+      <Bento.Visual>
+        <img src={placeholderImage} alt="random" />
+      </Bento.Visual>
+    </Bento.Item>
+  </Bento>
+)
+
+export const Example: StoryFn<typeof Bento> = () => (
+  <>
+    <style>
+      {`.purple-background {
+                background: linear-gradient(90deg, #6C5DD3 0%, #613DB4 100%);
+    	}
+		.block-icon {
+			display: block !important;
+		}
+		`}
+    </style>
+    <Bento>
+      <Bento.Item columnSpan={7} rowSpan={4} colorMode="dark" visualAsBackground className="purple-background">
+        <Bento.Content padding="normal" fixedBottomLink leadingVisual={<CopilotIcon />}>
```

Comment:
There's no spacing between the leading visual and heading, which looks quite odd. Is that per the design?

![Screenshot 2023-08-24 at 09 27 41](https://github.com/primer/brand/assets/13340707/7820aed0-3984-48a3-8975-1a0a3efc98f5)

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/390/commits/53fb31450e93b3c0e04a852af0a50db660c2c78a